.dockerignore

```diff
@@ -1 +1,8 @@
 test
+.cirecleci
```

Comment:
I probably could look this up myself, but as I'm here: does dockerignore support inverted statements, i.e. can we ignore everything but the source files (which is easier to keep track of)?

Reply:
Unfortunately, the exception modifier doesn't support wildcards for directories like * or ** which means, that we need to manually specify the folder structure and therefore also the single backends as they have a respective folder. Explanation of that is also in the .dockerignore as comments.

Reply:
It could have been so easy. Thanks for finding out.